app/javascript/packages/document-capture/components/optional-questions.scss

```diff
@@ -0,0 +1,24 @@
+@use 'uswds-core' as *;
+
+.document-capture-optional-questions {
+  margin-top: 2em;
+  .usa-tag {
+    background-color: #306f84ff;
+    color: white;
+  }
```

Comment:
You should be able to use the "Informative" variant of the tag instead of implementing custom styles here.

e.g.

```html
<span class="usa-tag usa-tag--informative">Informative</span>
```

https://federalist-340d8801-aa16-4df5-ad22-b1e3e731098e.sites.pages.cloud.gov/site/18f/identity-design-system/tags/

Reply:
@aduth fixed.